python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -188,11 +191,11 @@ def makeIdFactory(self, dataRef):
                              (e.g. visit, raft, sensor for LSST data).
         """
         expBits = dataRef.get("ccdExposureId_bits")
-        expId = long(dataRef.get("ccdExposureId"))
+        expId = int(dataRef.get("ccdExposureId"))
```

Comment:
We use 64 bits now, so `int` is safe (all IDs fit in 64 bits) so I propose to leave it as is.